Index: src/PkgFunctions.cc
===================================================================
--- src/PkgFunctions.cc	(revision 64337)
+++ src/PkgFunctions.cc	(working copy)
@@ -373,8 +373,6 @@
     ret->add(YCPString("credentials_global_dir"), YCPString(zconfig.credentialsGlobalDir().asString()));
     ret->add(YCPString("credentials_global_file"), YCPString(zconfig.credentialsGlobalFile().asString()));
 
-    ret->add(YCPString("plugins_path"), YCPString(zconfig.pluginsPath().asString()));
-
     ret->add(YCPString("distro_version_pkg"), YCPString(zconfig.distroverpkg()));
 
     return ret;
Index: src/Package.cc
===================================================================
--- src/Package.cc	(revision 64337)
+++ src/Package.cc	(working copy)
@@ -2107,9 +2107,6 @@
 
     switch (d_mode)
     {
-	case zypp::DownloadDefault:
-	    mode = "default";
-	    break;
 	case zypp::DownloadOnly:
 	    mode = "download_only";
 	    break;
