Index: src/Package.cc
===================================================================
--- src/Package.cc	(revision 66348)
+++ src/Package.cc	(revision 66349)
@@ -1812,8 +1812,6 @@
 	{
 	    y2milestone("Resetting the solver");
 	    zypp_ptr()->resolver()->reset();
-	    // reset also the dist upgrade flag (set by PkgUpdateAll())
-	    zypp_ptr()->resolver()->setUpgradeMode(false);
 	}
     }
 
